### PR TITLE
Add support for XHTML documents

### LINKED ISSUE #575: Making popper.js XHTML-friendly
popper.js 1.12.9 doesn't support XHTML, I mean W3C-validated XHTML running in XHTML mode, yet.

To support XHTML-mode, I modified popper.js source file, replaced all `.nodeName`  with `.nodeName.toUpperCase()`, and finally my Bootstrap dropdown did work.

Any plans to add XHTML support?

### PR DESCRIPTION
**Pull request by ExE-Boss**
Fixes #575.

### CODE REVIEW

File: packages/popper/src/modifiers/computeStyle.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ export default function computeStyle(data, options) {
   if (sideA === 'bottom') {
     // when offsetParent is <html> the positioning is relative to the bottom of the screen (excluding the scrollbar)
     // and not the bottom of the html element
-    if (offsetParent.nodeName === 'HTML') {
+    if (offsetParent.nodeName.toLowerCase() === 'html') {
```
Comment (FezVrasta) [inline packages/popper/src/modifiers/computeStyle.js]: I know it's just a detail but I'd rather keep the strings uppercase, may you switch these to use `.toUpperCase()` instead?

**Comment by ExE-Boss** [comment]
@FezVrasta I’ve addressed your review comment, so can this be merged now?

**Comment by FezVrasta** [comment]
I need to find some time to fix the build first, sorry :-(

**Comment by FezVrasta** [comment]
@ExE-Boss may you rebase on master? We got green light now

**Comment by FezVrasta** [comment]
Tests are still failing, do you have time to look into them please?

**Pull request closed without merging**